tests/functional/prometheous/test_prometheus_validations.py

```diff
@@ -974,3 +975,82 @@ def test_restart_prometheus_glusterfs_pod(self):
             # Try to fetch metric from prometheus pod
             self._fetch_metric_from_promtheus_pod(
                 metric='heketi_device_brick_count')
+
+    @pytest.mark.tier2
+    def test_metrics_workload_on_prometheus(self):
+        """Validate run metrics workload on prometheus"""
+
+        self.prefix = "autotest-{}".format(utils.get_random_str())
+
```

Comment:
self._registry_project_name = self.registry_sc.get(
            'restsecretnamespace')

Reply:
For this test, take project name from config as mentioned above